alf/algorithms/sac_algorithm.py

```diff
@@ -359,8 +359,9 @@ def _alpha_train_step(self, log_pi):
         info = SacAlphaInfo(
             loss=LossInfo(
                 loss=alpha_loss,
-                extra=dict(alpha_loss=alpha_loss, alpha=self._log_alpha.
-                           exp())))
+                extra=dict(
+                    alpha_loss=alpha_loss,
+                    alpha=self._log_alpha.exp().repeat(alpha_loss.shape[0]))))
```

Comment:
use expand() instead. expand does not copy. And we should add a comment somewhere about the shapes of the tensor in info.